build.py

```diff
@@ -1,15 +1,17 @@
 import os
 import subprocess
 
-vivado = "/opt/Xilinx/Vivado/2023.1/bin/vivado"
+xvlog = "/opt/Xilinx/Vivado/2023.1/bin/xvlog"
+xelab = "/opt/Xilinx/Vivado/2023.1/bin/xelab"
+xsim = "/opt/Xilinx/Vivado/2023.1/bin/xsim"
 
 class VivadoBuildError(Exception): pass
 
-if not os.access("build.tcl", os.R_OK):
+if not os.access("xsim_run.tcl", os.R_OK):
 	raise VivadoBuildError("you should pass a build script for us to run!")
 
 print("starting vivado, please wait...")
-proc = subprocess.Popen(f"{vivado} -mode batch -source build.tcl",
+proc = subprocess.Popen(f"{xvlog} --sv ./hdl/*",
```

Comment:
Now `build.py` is specific to running your testbench? We won't want to commit to main like this, but fine for testing on the branch.